auditlog/diff.py

```diff
@@ -2,7 +2,11 @@
 from django.core.exceptions import ObjectDoesNotExist
 from django.db.models import NOT_PROVIDED, DateTimeField, Model
 from django.utils import timezone
-from django.utils.encoding import smart_text
+
+try:
+    from django.utils.encoding import smart_str
+except ImportError:
+    from django.utils.encoding import smart_text as smart_str
```

Comment:
Why you don't import `smart_str` directly like:

```suggestion
from django.utils.encoding import smart_text
```